tests/README.md

```diff
@@ -1,10 +1,10 @@
-## Jenkins Server
-Here are Jenkins servers for CI
+## TEST env
```

Comment:
The title makes confuse developer,  because this markdown describe only for Jenkins CI.
Please change title or description.

Also, Plz change the link in repo root `/README.md` .
https://github.com/blue-oil/blueoil/blob/06d9a8222724e63f1326c41c4a8a9fff2e51d4e7/README.md#tests-for-ci